### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by caizhongang** [open-mmlab/mmhuman3d#48] [Announcements]
### Updates
- `2022-01-04` [v0.4.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.4.0) released. Check out our roadmap to [v0.5.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-1903687)!
- `2022-02-12` [v0.5.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.5.0) released. Check out our roadmap to [v0.6.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2157303)!
- `2022-04-01` [v0.6.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.6.0) released. Check out our roadmap to [v0.7.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2485542)!
- `2022-04-30` [v0.7.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.7.0) released. Check out our roadmap to [v0.8.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2691139)!
- `2022-06-01` [v0.8.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.8.0) released. Check out our roadmap to [v0.9.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2861960)!
- `2022-07-13` [v0.9.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.9.0) released. Check out our roadmap to [v0.10.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-3142467)!

### Backlog
More updates in the backlog (to be included in future releases):
- Support point cloud-based methods
- Collaboration with MMRazor and MMDeploy for mobile deployment
- Support generation of init params for custom datasets
- Support backbone feature extraction for custom datasets
- Save inference output in `fbx`
- Support methods for accurate shape estimation
- Support temporal fitting methods

### Links
Please add a comment in [wishlist](https://github.com/open-mmlab/mmhuman3d/discussions/47) if you have any feature request!

**Comment by caizhongang** [comment]
Roadmap towards v0.5.0 (released on `2022-02-12`, [full release notes](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.5.0)):
- [x] Support new data structure SMC for new dataset HuMMan that will soon be released
- [x] Support for multi-GPU training/testing without slurm
- [x] Support training-time validation and additional metrics such as PVE
- [x] Bug fixes in data augmentation for more stable training
- [x] Stronger HybrIK baseline (PA-MPJPE 49.02 on 3DPW)

Some features have been delayed to [v0.6.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2157303):
- Refactor differentiable renderers and support UV map rendering

**Reply by socketing** [reply]
我想知道mmhuman3d得到的参数模型是1:1吗，可以把模型分离出来并且利用参数化模型测量到人体的真实高度吗
Update: discussion moved to #80 .

**Comment by caizhongang** [comment]
Roadmap towards v0.6.0 (released on `2022-04-01`, [full release notes](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.6.0))
- [x] Add HumanDataCache that requires 96% less RAM during training
- [x] Refactor differentiable renderers and support UV map rendering
- [x] Support slice/concat operations for HumanData

**Comment by caizhongang** [comment]
Roadmap towards v0.7.0 (released on `2022-04-30`, [full release notes](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.7.0))
- [x] Support [PARE](https://pare.is.tue.mpg.de/) (better than the official implementation)
- [x] Support [DeciWatch](https://ailingzeng.site/deciwatch) (added by paper authors)
- [x] Add [GTA-Human](https://caizhongang.github.io/projects/GTA-Human/) HMR baseline (official release)
- [x] Support saving inference results

**Comment by caizhongang** [comment]
Roadmap towards v0.8.0 (released on `2022-06-01`, [full release notes](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.8.0))
- [x] Support [SmoothNet](https://ailingzeng.site/smoothnet) (added by paper authors)
- [x] Fix circular import and up to 2.5x speed up in module initialization
- [x] Add documentations in Chinese

Features that are delayed to [v0.9.0](https://github.com/open-mmlab/mmhuman3d/discussions/48#discussioncomment-2861960):
- Support [ROMP](https://github.com/Arthur151/ROMP)
- Support [ExPose](https://expose.is.tue.mpg.de/)

**Comment by caizhongang** [comment]
Roadmap towards v0.9.0 (released on `2022-07-13`, [full release notes](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.9.0))
- [x] Support [ExPose](https://expose.is.tue.mpg.de/)
- [x] Support new body model [STAR](https://star.is.tue.mpg.de/)
- [x] Release of [GTA-Human](https://caizhongang.github.io/projects/GTA-Human/) dataset with more baselines (SPIN-FT and PARE-FT)
- [x] Refactor registration and improve performance of SPIN

Features the are delayed to [v0.10.0]():
- [ ] Support [ROMP](https://github.com/Arthur151/ROMP)
- [ ] Upgrade HumanData to support multi-person/video data

**Comment by caizhongang** [comment]
Roadmap towards v0.10.0
- [ ] Support [ROMP](https://github.com/Arthur151/ROMP)
- [ ] Support [BEV](https://github.com/Arthur151/ROMP)
- [ ] Support [PIXIE](https://pixie.is.tue.mpg.de/)
- [ ] Release of [HuMMan](https://caizhongang.github.io/projects/HuMMan/) dataset and toolbox
- [ ] Upgrade HumanData to support multi-person/video data

**Reply by akk-123** [reply]
when will release v0.10.0? @caizhongang
